specutils/tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import requests
+import os
+
+import tempfile
+
+
+@pytest.fixture(scope='module')
+def remote_data_path(request):
+    """
+    Remotely access the Zenodo deposition archive to retrieve the versioned
+    test data.
+    """
+    file_id, file_name = request.param.values()
+
+    f = requests.get("https://zenodo.org/record/{}/files/{}?download=1".format(
```

Comment:
(Also, *very* minor nitpick: the variable shouldn't be `f` because it's not a file.  `res` is what I often use because it's the response to a request)

Reply:
I really like `requests` because it's the only package to implement a fully RESTful API, has a built-in json decoder, supports connection pooling, browser-style ssl verification, timeouts, thread safety, etc, etc. The list goes on. But I realize that that's not really necessary here, and the built-in library is probably sufficient.